index.html

```diff
@@ -0,0 +1,65 @@
+<!-- Este es un comentario -->
+
+<!DOCTYPE html>
+<html lang="es">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mi CV</title>
+</head>
+
+<body>
+  <header>
+    <h1>Sarah Ines Sole Villalobos</h1>
+   </header>
+    
+    <div>
+      <p>E-mail: sisv170906@gmail.com</p>
+      <p>Tel: +57 312 768 7070</p>
+      <p>Dir: Calle 152A #54-37, Bogotá, Colombia</p>
+    </div>
+
+  <header>
+    <h2>Formación academica</h2>
+  </header>
+
+  <div> 2024/actualidad: Ingenieria informatica </div>
+  <div> Universidad de la sabana </div>  
+  <div> 2023: Bachiller </div>
+  <div> Colegio Nueva Zelandia IED </div>
```

Comment:
Esto debería ser una lista y, en lugar de usar "div", usar "p". Los div únicamente se utiizan para agrupar contenido que posteriormente se va a estilizar, no es correcto que el hijo de un "div" sea directamente el texto. Lo más adecuado es que el texto siempre sea descendiente directo de un "p"

Se sugiere usar una lista dentro de otra lista para dar el efecto de indentación.

```suggestion
<ul>
   <li>
     <p>2024/actualidad: Ingeniería informática </p>
      <ul>
         <li><p>Universidad de La Sabana  </p></li>
      </ul>  
   </li>
   
   ...

</ul>
```